samples/openapi3/server/petstore/go/go-petstore/go/api_pet.go

```diff
@@ -270,10 +271,15 @@ func (c *PetAPIController) UploadFile(w http.ResponseWriter, r *http.Request) {
 	
 	
 	additionalMetadataParam := r.FormValue("additionalMetadata")
-	fileParam, err := ReadFormFileToTempFile(r, "file")
-	if err != nil {
-		c.errorHandler(w, r, &ParsingError{Err: err}, nil)
-		return
+	var fileParam *os.File
+	{
+		param, err := ReadFormFileToTempFile(r, "file")
+		if err != nil {
+			c.errorHandler(w, r, &ParsingError{Err: err}, nil)
+			return
+		}
+
+		fileParam = param
 	}
```

Comment:
I understand that this is uncommon, but this is purely cosmetic and does not affect the semantics of the code. 

This will need to be fixed with the larger problem as stated above https://github.com/OpenAPITools/openapi-generator/pull/17051#issuecomment-1814783198